tests/benchdnn/inputs/matmul/test_matmul_smoke

```diff
@@ -33,3 +33,19 @@
 --attr-post-ops=,linear:2:1
 --runtime_dims_masks=1:0,3:3
 --batch=shapes_2d_ci
+
+# Decompression
```

Comment:
these do not seem to cover decompression as fp-mathmode is not set here. Am I missing something?